sources/packages/api/src/database/migrations/1610712571660-User.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
+import { getSQLFileData } from '../../utilities';
+
+export class User1610712571660 implements MigrationInterface {
```

Comment:
This is a generated class name? if yes, why should we add it to source code? if no, why is the name of the class such complicated one?

Reply:
I think we already discussed, this template is generated by TypeORM utility. This is the template/format typeorm recognise for migration script. For more info check 
https://typeorm.io/#/migrations